torch/ao/quantization/_dbr/auto_trace.py

```diff
@@ -38,9 +39,12 @@ def add_auto_observation(
     qconfig_dict: Dict[str, Any],
     example_inputs: Tuple[Any],
     input_dtypes: Any = (torch.float,),  # must be same structure as model inputs
-    output_dtypes: Any = (torch.float,),  # must be same structure as model outputs
     prepare_custom_config_dict: Dict[str, Any] = None,
 ) -> torch.nn.Module:
+    if prepare_custom_config_dict is None:
+        prepare_custom_config_dict = {}
+    output_dtypes = prepare_custom_config_dict.get('output_dtypes', (torch.float,))
```

Comment:
is this a refactor? not sure where is this used

Reply:
this is to unbreak the `test_inplace_add` case.  I believe the FX comparison currently has the FX graph mode model return a long dtype (which might be a bug in FX graph mode quantization).  This code just makes sure the test keeps passing.